src/content/docs/accounts/accounts/automated-user-management/roles-permissions-automated-user-management.mdx

```diff
@@ -21,7 +21,7 @@ When using [automated user management](/docs/accounts/accounts/automated-user-ma
 An example of how the user groups in your identity provider might map over to [access grants](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#understand-concepts) in New Relic, which assign a group access to a specific role (default or customer) on a specific account. 
 </figcaption>
 
-In the above example, let’s focus on the Engineers group. Via SCIM, New Relic will receive the two user records as well as the information that they belong to the Engineers group. In this example, the configuration would be: the two Engineer users have the [**Read only** role](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model#standard-roles) on Account 1, the [**Standard user**](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model#standard-roles) role on Account 2, and the [**All product admin**](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model#standard-roles) role on Account 3.
+In the above example, let’s focus on the Engineers group. Via SCIM, New Relic will receive the two user records as well as the information that they belong to the Engineers group. In this example, the configuration would be: the two Engineer users have the [**Read only** role](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#roles) on Account 1, the [**Standard user**](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#roles) role on Account 2, and the [**All product admin**](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#roles) role on Account 3.
```

Comment:
@zuluecho9, all three of the links in this paragraph failed for me locally.

Reply:
that's weird. fwiw, I don't even check stuff locally any more, I just check the gatsby build here. too many issues with local build.